SQL/0000-00-01-Permission.sql

```diff
@@ -107,7 +107,8 @@ INSERT INTO `permissions` VALUES
     (45,'issue_tracker_reporter', 'Can add a new issue, edit own issue, comment on all', 2),
     (46,'issue_tracker_developer', 'Can re-assign issues, mark issues as closed, comment on all, edit issues.', 2),
     (47,'imaging_browser_phantom_allsites', 'Can access only phantom data from all sites in Imaging Browser', 2),
-    (48,'imaging_browser_phantom_ownsite', 'Can access only phantom data from own site in Imaging Browser', 2);
+    (48,'imaging_browser_phantom_ownsite', 'Can access only phantom data from own site in Imaging Browser', 2),
+    (49,'instrument_manager', 'Instrument Manager: View module and upload instruments', 2);
```

Comment:
I think there should be 2 permissions here.. one to see the list of instruments that are installed, and one to upload a new instrument.

Reply:
Done!